packages/webview_flutter/webview_flutter_wkwebview/darwin/Classes/FWFWebViewConfigurationHostApi.m

```diff
@@ -95,21 +95,25 @@ - (void)setAllowsInlineMediaPlaybackForConfigurationWithIdentifier:(NSInteger)id
                                                              error:
                                                                  (FlutterError *_Nullable *_Nonnull)
                                                                      error {
+#if TARGET_OS_IOS
   [[self webViewConfigurationForIdentifier:identifier] setAllowsInlineMediaPlayback:allow];
+#endif
+  // No-op, rather than error out, on macOS, since it's not a meaningful option on macOS and it's
+  // easier for clients if it's just ignored.
```

Comment:
do we want to just `NSLog` here in case of confusion?

Reply:
In practice, people are going to be calling this with `YES`, which is already how macOS works, so it would be a spammy warning that wouldn't really mean anything.